### PR TITLE
Delete docs/book/src/TODO.md

### PR DESCRIPTION
**Pull request by pierreozoux**

### CODE REVIEW

File: docs/book/src/TODO.md [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# TODO
```
Comment (camilamacedo86) [inline docs/book/src/TODO.md:1]: What is your motivation to remove this one?

Comment (pierreozoux) [reply docs/book/src/TODO.md:1]: this file is a plceholder that is not used yet, just that :)

Comment (camilamacedo86) [reply docs/book/src/TODO.md:1]: It is used when we face 404, see: https://github.com/kubernetes-sigs/kubebuilder/blob/ce24b89a23fb8a51be78b115c15fd0f08975b46a/docs/book/src/404.md?plain=1

Comment (camilamacedo86) [reply docs/book/src/TODO.md:1]: As other places see; https://github.com/search?q=repo%3Akubernetes-sigs%2Fkubebuilder%20TODO.md&type=code

**Pull request closed without merging**